README.md

```diff
@@ -24,7 +24,7 @@ using many of the same open source packages that Google developers are using
 in their everyday work.
 
 To get started with the JAX AI stack, you can check out [Getting started with JAX](
-https://github.com/jax-ml/jax-ai-stack/blob/main/docs/source/getting_started_with_jax_for_AI.ipynb).
+https://github.com/jax-ml/jax-ai-stack/blob/main/docs/source/getting_started.md).
```

Comment:
Thanks, but we should link to the rendered page rather than the markdown source: https://docs.jaxstack.ai/en/latest/getting_started.html

Reply:
Updated the link now to the rendered page. I wasn't sure if we wanted this since the original link was the jupyter notebook.